shader/flatten.wgsl

```diff
@@ -200,7 +217,7 @@ fn es_seg_from_params(p0: vec2f, p1: vec2f, params: EulerParams) -> EulerSeg {
 
 fn es_seg_eval_with_offset(es: EulerSeg, t: f32, offset: f32) -> vec2f {
     let chord = es.p1 - es.p0;
-    let scaled = offset / length(chord);
+    let scaled = offset / max(length(chord), TANGENT_THRESH);
```

Comment:
Not sure if this should be capped at `sqrt(TANGENT_THRESH)` instead of `TANGENT_THRES`.